etc/config/c++.defaults.properties

```diff
@@ -1,5 +1,5 @@
 # Default settings for C++
-compilers=/usr/bin/g++-4.4:/usr/bin/g++-4.5:/usr/bin/g++-4.6:/usr/bin/g++-4.7:/usr/bin/g++-4.8:/usr/bin/clang++/usr/bin/g++-4.8:/usr/bin/g++
+compilers=/usr/bin/g++-4.4:/usr/bin/g++-4.5:/usr/bin/g++-4.6:/usr/bin/g++-4.7:/usr/bin/g++-4.8:/usr/bin/clang++:/usr/bin/g++-5:/usr/bin/g++-7:/usr/bin/g++-6:/usr/bin/g++
```

Comment:
Also, I don't know if doing this in this PR would be a good idea (That's on @mattgodbolt) but this looks like something we *should* do in another PR as this changes the default behaviour of CE

Reply:
good point!